specs/BadgeNotifications/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,192 @@
+| NewMessage | A glyph that indicates the arrival of a new message, often used in messaging and email applications. | ![Screenshot](badge-newmessage.png) |
+| Paused | A glyph that signifies a pause in activity or content, such as media playback being paused. | ![Screenshot](badge-paused.png) |
+| Playing | A glyph that indicates media or content is currently playing. | ![Screenshot](badge-playing.png) |
+| Unavailable | A glyph that shows the user or a service is not currently available or offline. | ![Screenshot](badge-unavailable.png) |
+| Error | A glyph that denotes an error has occurred, which may require user action to resolve. | ![Screenshot](badge-error.png) |
+
+These glyphs are system-provided and standardized, ensuring a consistent look and feel across different applications that use them. By using these glyphs, developers can convey specific states or notifications to users in a visually intuitive manner.
+
+# API Details
+
+> Note: all of this new WinAppSDK API is to support 
+  Badge Notifications in WindowsAppSdk
+ 
+```c++ (but really MIDL3)
+namespace Microsoft.Windows.BadgeNotifications
+{
+    // Set of predefined glyphs that can be used to represent various statuses or notifications on an application's badge
+    enum BadgeGlyphValue
+    {
+        None, // No glyph. A blank tile appears in the badge.
+        Activity, // A glyph representing an activity
+        Alert, // A glyph representing an alert
+        Alarm, // A glyph representing an alarm
+        Attention, // A glyph representing attention
+        Available, // A glyph representing availability
+        Away, // A glyph representing being away
+        Busy, // A glyph representing being busy
+        NewMessage, // A glyph representing a new message
+        Paused, // A glyph representing being paused
+        Playing, // A glyph representing playing
+        Unavailable, // A glyph representing being unavailable
+        Error, // A glyph representing an error
+    };
+
+    runtimeclass BadgeNotification
+    {
+        // Constructs a BadgeNotification with a numeric badge value.
+        // This constructor initializes the BadgeNotification with a number that will be displayed on the badge.
+        // @param num The numeric value to be displayed on the badge.
+        BadgeNotification(UInt32 num);
+
+        // Constructs a BadgeNotification with a glyph value.
+        // This constructor initializes the BadgeNotification with a predefined glyph from the BadgeGlyphValue enum.
+        // @param value The BadgeGlyphValue representing a system-provided glyph to be displayed on the badge.
+        BadgeNotification(BadgeGlyphValue value);
+
+        // Gets or sets the time after which a Badge Notification should not be displayed.
+        Windows.Foundation.DateTime Expiration;
+    }
+
+    // The manager class which encompasses all Badge Notification API Functionality
+    runtimeclass BadgeNotificationManager
```

Comment:
What happens when one app has two top-level windows that don't "glom together" - can each top-level window get its own badge? Or is it applied to only the "main" window (whatever that is)?

Reply:
It is only applied to the "main" window. Verified in the current behavior as well. When you create a new window of the outlook app only the main windows have the badge notification